src/locales/en_JP/index.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * This file is automatically generated.
+ * Run 'pnpm run generate:locales' to update.
+ */
+import type { LocaleDefinition } from '../..';
+import name_ from './name';
+
+const en_JP: LocaleDefinition = {
+  title: 'English Names (Japan)',
```

Comment:
```suggestion
  title: 'English (Japan)',
```

@Shinigami92 What do you think?

Reply:
```suggestion
  title: 'English (Japan)',
```